documentation/planning.md

```diff
@@ -50,18 +49,34 @@ So each LED per zone will get a number from 0 to n, where n is the last LED in t
-They are all fully supported by TesLight.
+### Luminant Types
+Currently there are three different types of luminants available that you can choose from. They are all fully supported by TesLight, but not each luminant is suitable for each installation location.
+
+#### Basic WS2812 LED strip 
+This is the most widely availble type of luminant - a series of WS2812 LEDs in mounted in series on a thin, flexible PCB strip. These strips are available in various lenghts, widths and density (count of LEDs per meter). 
+Possible installation locations for the LED strip are the dashboard and the foot wells. A density of around 60 LEDs/meter is a good compromise between cost/power consumption and good looking light (effects).
+Benefits of the LED strip are that it is easy to obtain in various configurations and since many LEDs are distributed across the strip it can be used for many light effects.
+The main drawback is that it can only be installed in locations where it is not directly visible.
+
+#### Fiber tube with WS2812 LEDs as light injectors
+This type of luminant is a small flexible tube with a tab on it's side. WS2812 LEDs can be placed on one or (better) both ends of the tube to inject light into the tube and make it glow (fairly) evenly along its entire length.
+The fiber tube is suitable for all installation locations in the Tesla Model 3/Y with the exception of the footwell lighting. As of now it is also the only lumimant that can be installed in the 2021 or later center console of Model 3/Y and around the seats.
+Benefits of the fiber tube are the low cost, low power consumption and easy, flexible installation.
+The main drawback of the fiber tube is that you can have at most 2 colors per tube segment, one injected from each side, which greatly limits the effects that you can display with it. Also, because of it's white(-ish) color the fiber tube visually stands out a bit againts the dark interior in daylight which can look less professional depending on your taste.
 
-- Use a basic WS2812 LED strip
-- Use a fiber tube and 2 WS2812 LEDs as light injectors
-- Use a light bar which looks cleaner than a fiber tube and allows awesome animations
+#### Light bar with WS2812 LEDs 
+The light bar is like a hybrid of the light strip and fiber tube. It consists of a very thin WS2812 LED strip mounted onto an also very thin and flexible light distribution bar that is additionally covered in a black, translucent tape. The density of the currently available light bars is very high with around 130 LEDs/meter). Possible installation locations are the dashboard, the door panels and the pre-2021 center console.
+Benefits: Because of its size and flexibility it can be installed almost anywhere in the care, where a fiber tube can be installed as well. But since it has distributed LEDs all along its length and not just at the ends it supports way more effects than the fiber tube. Because it is wrapped in black type it is also not very visible in daylight.
```

Comment:
"Because it is wrapped in black type it is also not very visible in daylight.": This sounds like a dawaback, but is indeed a benefit, so barely not visible when not turned on or something like that would be more precise.